.isort.cfg

```diff
@@ -6,5 +6,6 @@ line_length=100
 indent=2
 lines_after_imports=2
 known_first_party=internal_backend,pants,pants_test
+known_standard_library=_dummy_thread,_markupbase,_thread,builtins,copyreg,html,http,queue,reprlib,socketserver,tkinter,xmlrpc
```

Comment:
Please add a TODO to remove this pending https://github.com/pantsbuild/pants/issues/6149